content/id/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Memperluas Kubernetes API dengan lapisan agregasi
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Lapisan agregasi memungkinkan kubernetes untuk diperluas dengan API tambahan, di luar dari apa yang ditawarkan oleh inti API kubernetes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Ikhtisar
+Lapisan agregasi memungkinkan menginstall tambahan API _Kubernetes-style_ di cluster anda. Ini bisa berupa bawaan, maupun solusi dari pihak ke-3 yang sudah ada, seperti [layanan-katalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/README.md), atau API yang telah dibuat user seperti [pembangun-apiserver](https://github.com/kubernetes-incubator/apiserver-builder/blob/master/README.md), yang dapat membantu Anda memulai.
+
+Lapisan agregasi berjalan di dalam proses dengan kube-apiserver. Hingga sebuah sumber daya ekstensi terdaftar, Lapisan agregasi tidak akan melakukan apapun. Untuk mendaftarkan sebuah API, pengguna harus menambahkan objek _APIService_, yang "mengklaim" jalur URL di API Kubernetes. Di poin ini, lapisan agregasi akan mem-_proxy_ apapun yang dikirim ke jalur API tersebut (misalnya /apis/myextension.mycompany.io/v1/…) ke _APIService_ yang terdaftar. 
+
+Biasanya, _APIService_ akan diimplementasikan oleh sebuah ekstensi-apiserver di _pod_ yang berjalan di cluster. ekstensi-apiserver ini biasanya perlu di pasangkan dengan satu atau lebih _controller_ apabila manajemen aktif dari sumber daya tambahan diperlukan, pembuat-apiserver sebenarnya akan memberikan kerangka untuk keduanya. Sebagai contoh lain, ketika layanan-katalog diinstal, ia menyediakan ekstensi-apiserver dan _controller_ untuk layanan yang disediakannya.
```

Comment:
all API objects should have camelcase format
```suggestion
Biasanya, _APIService_ akan diimplementasikan oleh sebuah ekstensi-apiserver di Pod yang berjalan di kluster. ekstensi-apiserver ini biasanya perlu di pasangkan dengan satu atau lebih _controller_ apabila manajemen aktif dari sumber daya tambahan diperlukan, pembuat-apiserver sebenarnya akan memberikan kerangka untuk keduanya. Sebagai contoh lain, ketika layanan-katalog diinstal, ia menyediakan ekstensi-apiserver dan _controller_ untuk layanan yang disediakannya.
```

Reply:
Addressed here https://github.com/kubernetes/website/pull/16903/commits/66b6dd5d7bfd3e27483ba07ac840bfeaef370996